ksql-rest-app/src/test/resources/ksql-plan-schema/schema-final.json

```
@@ -0,0 +1,885 @@
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamFilterV1" ],
+          "default" : "streamFilterV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "filterExpression" : {
+          "type" : "string"
+        }
+      },
+      "title" : "streamFilterV1",
+      "required" : [ "@type", "properties", "source", "filterExpression" ]
+    },
+    "StreamFlatMap" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamFlatMapV1" ],
+          "default" : "streamFlatMapV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "tableFunctions" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        }
+      },
+      "title" : "streamFlatMapV1",
+      "required" : [ "@type", "properties", "source", "tableFunctions" ]
+    },
+    "StreamGroupBy" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamGroupByV1" ],
+          "default" : "streamGroupByV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "formats" : {
+          "$ref" : "#/definitions/Formats"
+        },
+        "groupByExpressions" : {
+          "type" : "array",
+          "items" : {
+            "type" : "string"
+          }
+        }
+      },
+      "title" : "streamGroupByV1",
+      "required" : [ "@type", "properties", "source", "formats", "groupByExpressions" ]
+    },
+    "StreamGroupByKey" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamGroupByKeyV1" ],
+          "default" : "streamGroupByKeyV1"
+        },
+        "properties" : {
+          "oneOf" : [ {
+            "$ref" : "#/definitions/DefaultExecutionStepProperties"
+          } ]
+        },
+        "source" : {
+          "$ref" : "#/definitions/ExecutionStep"
+        },
+        "formats" : {
+          "$ref" : "#/definitions/Formats"
+        }
+      },
+      "title" : "streamGroupByKeyV1",
+      "required" : [ "@type", "properties", "source", "formats" ]
+    },
+    "StreamMapValues" : {
```

Consider:

```suggestion
    "StreamSelect" : {
```

Given that's what the step is actually performing.  `mapValues` just happened to be the KS function name and actually no longer is, it's not using `transformValues`.